core/src/main/java/org/dozer/DozerBeanMapperBuilder.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2005-2017 Dozer Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dozer;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.dozer.classmap.ClassMapBuilder;
+import org.dozer.config.BeanContainer;
+import org.dozer.config.GlobalSettings;
+import org.dozer.factory.DestBeanCreator;
+import org.dozer.loader.CustomMappingsLoader;
+import org.dozer.loader.MappingsParser;
+import org.dozer.loader.xml.XMLParser;
+import org.dozer.loader.xml.XMLParserFactory;
+import org.dozer.osgi.Activator;
+import org.dozer.osgi.OSGiClassLoader;
+import org.dozer.stats.StatisticsManager;
+import org.dozer.stats.StatisticsManagerImpl;
+import org.dozer.util.DefaultClassLoader;
+import org.dozer.util.DozerClassLoader;
+import org.dozer.util.RuntimeUtils;
+
+/**
+ * Builds an instance of {@link DozerBeanMapper}.
+ * Provides fluent interface to configure every aspect of the mapper. Everything which is not explicitly specified
+ * will receive its default value. Please refer to class methods for possible configuration options.
+ */
+public final class DozerBeanMapperBuilder {
+
+    private List<String> mappingFiles = new ArrayList<>(1);
+    private DozerClassLoader classLoader;
+
+    private DozerBeanMapperBuilder() {
+    }
+
+    /**
+     * Creates new builder. All the configuration has its default values.
+     *
+     * @return new instance of the builder.
+     */
+    public static DozerBeanMapperBuilder create() {
+        return new DozerBeanMapperBuilder();
+    }
+
+    /**
+     * Creates an instance of {@link DozerBeanMapper}, with all the configuration set to its default values.
+     * <p>
+     * Is a shortcut for {@code DozerBeanMapperBuilder.create().build()}.
+     *
+     * @return new instance of {@link DozerBeanMapper} with default configuration.
+     */
+    public static DozerBeanMapper buildDefaultImplicit() {
```

Comment:
Consider renaming. Build default implicit is confusing. buildDefault probably is a better option.